sklearn/metrics/_plot/tests/test_common_curve_display.py

```diff
@@ -36,9 +36,7 @@ def test_display_curve_error_non_binary(pyplot, data, Display):
     X, y = data
     clf = DecisionTreeClassifier().fit(X, y)
 
-    msg = (
-        "Expected 'estimator' to be a binary classifier, but got DecisionTreeClassifier"
-    )
+    msg = "Expected 'estimator' to be a binary classifier."
```

Comment:
Showing only this part of the expected message makes the test confusing. I'd rather add the part with the classes